src/Monitor/Monitor/help/Add-AzMetricAlertRuleV2.md

```diff
@@ -0,0 +1,417 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Monitor.dll-Help.xml
+Module Name: Az.Monitor
+online version:
```

Comment:
@AnanyShah you will need to add a URL to the https://docs.microsoft.com file corresponding to each markdown file. The Jenkins build will fail due to this and notify you what the URLs are for each file.